end-to-end-test/local/runtime-config/run_container_screenshot_test.sh

```diff
@@ -15,6 +15,20 @@ docker run -it --rm \
     -v "$PORTAL_SOURCE_DIR:/cbioportal-frontend/" \
     $SCREENSHOT_IMAGE_NAME
 
+docker run -it --rm \
+    --name screenshot-e2e \
+    --network="$DOCKER_NETWORK_NAME" \
+    -e CBIOPORTAL_URL="$CBIOPORTAL_URL" \
+    -e SCREENSHOT_DIRECTORY="$SCREENSHOT_DIRECTORY"\
+    -e SPEC_FILE_PATTERN="./local/specs/core/**/ie.js" \
+    -e TEST_IE11="true" \
+    -e TEST_BROWSERSTACK="true" \
+    -e JUNIT_REPORT_PATH="$JUNIT_REPORT_PATH" \
+    -v "$PORTAL_SOURCE_DIR:/cbioportal-frontend/" \
+    $SCREENSHOT_IMAGE_NAME
```

Comment:
this seems duplicated with `run_container_ie_test.sh`

Reply:
I think having it in separate scripts would be cleaner e.g. in `run_container_ie_test.sh` as you had before? Now the localdb tests is not really as local anymore 

If it's too tricky to make these jobs separate I think we should update the documentation in the `README` to clarify how the localdb tests work?